build.gradle

```diff
@@ -136,6 +139,11 @@ subprojects {
         withJavadocJar()
     }
 
+    shadowJar {
+        archiveClassifier.set(null)
+        mergeServiceFiles()
```

Comment:
Not strictly required, but this combines all the SPI implementations available given the dependencies. So you could just declare the ddb-client dependency and be able to use the remote- or aws- client.